tests/integration/test_schema_backup.py

```diff
@@ -73,7 +75,12 @@ async def test_backup_restore(
             fp,
         )
 
-    config = set_config_defaults({"bootstrap_uri": kafka_servers.bootstrap_servers})
```

Comment:
This was a source of flakiness, the backup test would overwrite the `_schemas` topic used by other tests. With these changes each test gets its own topic so this doesn't happen.